app/utils/general.js

```diff
@@ -48,6 +49,27 @@ export function emptyFunction() { // eslint-disable-line no-empty-function
 
 }
 
+export const showTermsOfServiceModal = (navigator, theme) => {
+    MaterialIcon.getImageSource(Platform.OS === 'ios' ? 'chevron-left' : 'arrow-back', Platform.OS === 'ios' ? 32 : 20, theme.sidebarHeaderTextColor).then((source) => {
```

Comment:
All the modals in the app are using a close icon with an *X* is there a particular reason why that's not the case here?

Reply:
This is still a UX issue

Reply:
This was from the original design. https://docs.google.com/document/d/1VaXy4VD7eVzcwn5KpR8M5lGIVSt54_-iIgQXt7EePoQ/edit?disco=uiAAAACMw0IBw

https://projects.invisionapp.com/share/Q2NW58BFMTR#/screens/318715429

I don't know why this Modal has a `back` button and the rest of the Modals in the app have `close` buttons.
@jasonblais Thoughts?

Reply:
two options:
1. Change it to be a close icon and leave the current flow
2. Leave the icon and change the flow so this screen is pushed and not a modal (probably not the best way to go)

Reply:
That screen came from our UX design - if every other modal on mobile has a `close` button, then let's also have a `close` button here.

I.e. go with option 1 suggested by Elias.